### PR TITLE
When game is created, the deck is shuffled and hands are dealt to the…

### PR DESCRIPTION
**Pull request by elijahlong5**
… players.

### CODE REVIEW

**Comment by elijahlong5** [comment]
resolves #48

File: hanabi/game/models/games.py [hunk 1]
```diff
@@ -42,6 +45,19 @@ class Meta:
...
+        deck_length = 50
+        deck_order_values = list(range(deck_length))
+        random.shuffle(deck_order_values)
+        for player in self.players:
+            PlayerHand.create(player=player)
+
+            for i in range(0, Game.HAND_SIZE):
+                PlayerHandCard.create(
+                    card=self.cards.get(deck_order=deck_order_values.pop()),
+                    hand=self.players.get(player),
```
Comment (cdriehuys) [inline hanabi/game/models/games.py:58]: Isn't this supposed to refer to a hand rather than a player?

File: hanabi/game/models/games.py [hunk 2]
```diff
@@ -82,5 +99,5 @@ def create_from_lobby(cls, lobby):
                             deck_order=deck_order_values.pop(),
                             number=number,
                         )
-
+        cls.deal(game)
```
Comment (cdriehuys) [inline hanabi/game/models/games.py:102]: While innovative, this is more clearly written as: `game.deal()`.

**Review by cdriehuys — CHANGES_REQUESTED** [review]
This is a good start to dealing cards, but unfortunately there is a lot more logic here as a result of our decision to track individual game actions rather than keeping track of only the current state.

See #51 for the complete logic of dealing.

**Comment by cdriehuys** [comment]
This has been rebased and merged but GitHub went down in the middle of it.

**Pull request closed without merging**